### DISCUSSION TITLE
generic_oauth Provider "api_url"

### DISCUSSION THREAD

**Discussion by MB175** [grafana/grafana#58810] [General]
Hi, is somebody aware if the "api_url" (most commonly user-info endpoint of an oauth2 service), is called every login of a user.
Sometimes it looks like the api_url is only called when the user first logs-in to grafana.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
